source/configuration_manual/authentication/auth_policy.rst

```diff
@@ -0,0 +1,280 @@
+.. _authentication-auth_policy:
+
+Authentication policy support
+=============================
+
+.. versionadded:: v2.2.27
+
+Dovecot supports external authentication policy server. This
+server can be used to decide whether the connecting user is permitted,
+tarpitted or outright rejected. While dovecot can do tarpitting and
+refusal on its own, this adds support for making cluster-wide decisions
+to make it easier to deter and defeat bruteforce attacks.
+
+Known issues
+------------
+
+Prior v2.2.34, the request attributes contained **orig_username** which
+was not correct in all cases, especially with master login.
+
+Prior v2.3.5.2 / 2.2.36.3, invalid UTF-8 would crash auth server if auth
+policy is used.
+
+Configuration
+-------------
+
+The auth-policy server is a core feature and does not require plugin(s)
+to work. To activate this feature, you need to configure it.
+
+-  **auth_policy_server_url**: URL of the policy server, url is appended
```

Comment:
monospace? .. or not?.. I'm not entirely sure about these settings. At some point we should figure out what their formatting should be, but for now they seem to be rather inconsistent. So I guess either way for now.

Reply:
I did go for more monospace and just kept bold in very few placed where i think it's meant like it.